datanode/sqlsubscribers/position.go

```diff
@@ -112,11 +119,29 @@ func (p *Position) Push(ctx context.Context, evt events.Event) error {
 		return p.handleSettleMarket(ctx, event)
 	case tradeEvent:
 		return p.handleTradeEvent(ctx, event)
+	case ordersClosed:
+		return p.handleOrdersClosedEvent(ctx, event)
 	default:
 		return errors.Errorf("unknown event type %s", evt.Type().String())
 	}
 }
 
+func (p *Position) handleOrdersClosedEvent(ctx context.Context, event ordersClosed) error {
+	p.mutex.Lock()
+	defer p.mutex.Unlock()
+	// @TODO implement this
+	positions, err := p.store.GetByMarketAndParties(ctx, event.MarketID, event.Parties())
+	if err != nil {
+		return errors.Wrap(err, "error getting positions")
```

Comment:
we are a bit inconsistent on this so its not always obvious from looking at the code the correct approach is, but the agreement is that we should not use errors.Wrap any more (we should really remove all locations...still), so in this case:
fmt.Errorf("failed to get by market and parties: %w", err) or similar  (and yes its a maintenance PITA - but overall was agreed the best way)